tests/common/dualtor/dual_tor_mock.py

```diff
@@ -383,6 +382,9 @@ def apply_tunnel_table_to_dut(cleanup_mocked_configs, rand_selected_dut, mock_pe
             }
         }
     }
+    if 'spc' in rand_selected_dut.get_asic_name():
```

Comment:
Should we check if the device is `mlnx`?

Reply:
spc is a nvidia asic name. if you prefer device and not name we can replace.

Reply:
OK, the check is strict enough, thanks!

Reply:
OK, the check is strict enough, thanks